src/etl/ClinicalTrialModelFactory.ts

```diff
@@ -174,18 +174,18 @@ export class ClinicalTrialModelFactory {
       [this.unavailable],
       [new TherapeuticAreaModel(this.emptyIfNull(riphDmDto.domaine_therapeutique), this.unavailable)],
       new ContactDetailsModel(
-        this.emptyIfNull(riphDmDto.organisme),
-        this.emptyIfNull(riphDmDto.prenom),
-        this.emptyIfNull(riphDmDto.nom),
-        this.emptyIfNull(riphDmDto.adresse),
-        this.emptyIfNull(riphDmDto.ville),
-        this.emptyIfNull(riphDmDto.pays),
-        this.emptyIfNull(riphDmDto.code_postal),
+        this.emptyIfNull(riphDmDto.deposant_organisme),
```

Comment:
question: Pour ma culture, qu'est-ce qui fait que l'on n'implémente pas un objet pour ces attributs ? 
```suggestion
        this.emptyIfNull(riphDmDto.deposant.organisme),
        this.emptyIfNull(riphDmDto.deposant.prenom),
        this.emptyIfNull(riphDmDto.deposant.nom),
        this.emptyIfNull(riphDmDto.deposant.adresse),
        this.emptyIfNull(riphDmDto.deposant.ville),
        this.emptyIfNull(riphDmDto.deposant.pays),
        this.emptyIfNull(riphDmDto.deposant.code_postal),
```